tests/csm/rest/test_capacity_usage.py

```diff
@@ -366,18 +367,16 @@ def test_33919(self):
 
         self.log.info("[START] Recovery loop")
         failure_cnt = len(self.failed_pod)
-        for deploy_name in reversed(self.failed_pod):
-            self.log.info("[Start]  Restore deleted pods : %s", deploy_name)
-            resp = self.master.create_pod_replicas(num_replica=1, deploy=deploy_name)
+        for set_name, num_replica in self.failed_pod:
+            self.log.info("[Start]  Restore deleted pods : %s-%s", set_name, num_replica)
+            resp = self.ext_obj.restore_data_pod(set_name, num_replica)
             self.log.debug("Response: %s", resp)
-            assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
-            self.log.info("Successfully restored pod by %s way", self.restore_method)
-            self.failed_pod.remove(deploy_name)
-            self.log.info("[End] Restore deleted pods : %s", deploy_name)
+            assert resp, "Failed to restore pod"
```

Comment:
We may want to delete the entries from self.failed_pod

Reply:
added